pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -257,14 +255,13 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		updateCleanupPolicy = kubectl.RenameRollingUpdateCleanupPolicy
 	}
 	config := &kubectl.RollingUpdaterConfig{
-		Out:            out,
-		OldRc:          oldRc,
-		NewRc:          newRc,
-		UpdatePeriod:   period,
-		Interval:       interval,
-		Timeout:        timeout,
-		CleanupPolicy:  updateCleanupPolicy,
-		UpdateAcceptor: kubectl.DefaultUpdateAcceptor,
+		Out:           out,
+		OldRc:         oldRc,
+		NewRc:         newRc,
+		UpdatePeriod:  period,
+		Interval:      interval,
+		Timeout:       timeout,
+		CleanupPolicy: updateCleanupPolicy,
```

Comment:
So should we go ahead and make the availability type public and make it a field on RollingUpdaterConfig? The default would still be "created".